src/streamlink/plugins/livestream.py

```diff
@@ -16,7 +16,7 @@
 
 
 @pluginmatcher(re.compile(
-    r"https?://(?:www\.)?livestream\.com/"
+    r"https?://(?:www\.|api\.new\.)?livestream\.com/"
```

Comment:
I'll see when I have the time to rewrite the whole process with the API as a base URL for scraping, and, on the regex, match anything finishing with *livestream.com*.

This could permit to make both API and basic Livestream URLs being read.

I mainly made the change directly when I've seen that API links were the only endpoint for hidden channels.